### DISCUSSION TITLE
How to dynamiclly link library in deploy script?

### DISCUSSION THREAD

**Discussion by allan-avalon** [foundry-rs/foundry#4837] [Help]
I have a contract that uses 4 externally linked libraries when deployed. But I can't figure out how to link them (not changed, certain addresses) at compile time in the deploy script.
https://github.com/foundry-rs/foundry/discussions/1618 seems to have discussed the issue of linking external libraries using `forge create`, but I think it's different from what I want to ask, which is using `forge script`

**Comment by beeb** [comment]
No luck, heh? I have the same problem.
